.github/workflows/on_mr.yml

```diff
@@ -0,0 +1,22 @@
+name: MergeRequestChecker
+run-name: ${{ github.actor }} is running 🚀
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  hugochecker:
+    runs-on: node-20
+    steps:
+    - name: 'Check out the Hugo *.md files'
+      uses: ChrisPrusik/HugoChecker@v6
+      with:
+        hugo-folder: ${{ github.workspace }}
+  # fix_things1:
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - name: Check Spelling
+  #       uses: check-spelling/check-spelling@v0.0.22
```

Comment:
If you want to use check-spelling, you'd need to add `with:` / `checkout: true` or add a `- uses: actions/checkout@v4` step.

I'll add a warning message to suggest doing one or the other.

Reply:
Thanks!

Reply:
(You'll want to use v0.0.24; I've added the error -- it should be included in v0.0.25)

Reply:
This action is amazing @jsoref ! 
It gives precise information and pinpoints where issues are.